src/main/scala/org/scalajs/tools/tsimporter/sc/Definitions.scala

```diff
@@ -220,6 +220,20 @@ class MethodSymbol(nme: Name) extends Symbol(nme) with JSNameable {
 
   def paramTypes = params.map(_.tpe)
 
+  def modifiers: Modifiers = {
+    def noParams = tparams.isEmpty && params.isEmpty
+    def needsOverride = name match {
+      case Name("toString") => noParams && resultType == TypeRef.String
+      case Name("clone") => noParams // Any return type will trigger the error
```

Comment:
Vertically align the `=>` in one-line pattern matches.

Reply:
Also I think ` && resultType == TypeRef.String` shouldn't be there. Scala does not consider result types when deciding whether a method overrides another.

Reply:
You are right, aligned and removed `resultType` check from `toString`.